### DISCUSSION TITLE
[5.x] Move entry revisions from breadcrumbs into a dedicated sidebar area

### DISCUSSION THREAD

**Discussion by aaronbushnell** [craftcms/cms#14538] [Ideas]
While it's consistent with how Craft 4.x operates I don't think it makes sense to put entry revision information next to the title of the entry.

Since this UI is no longer just a title but is now a series of breadcrumbs decluttering this makes both of these interfaces a bit simpler and clearer for content editors.

Below is a _very rough_ example I created in Sketch after some in-browser tweaks.

I think this does a couple of important things as 5.0 is implementing significant UI changes:

1. It simplifies the behavior and look of the breadcrumbs
2. It prevents revisions from being hidden behind interactions
3. It grants a dedicated space for revisions so it can take on new capabilities beyond the context menu

![Screenshot 2024-03-05 at 4 41 29 PM](https://github.com/craftcms/cms/assets/315202/f219f773-0107-4a2a-ac1e-e7369b3cca31)

**Comment by wsydney76** [comment]
It's my retirement job right now to bring some useful helpers that have accumulated over the years to Craft 5 in a bundled form, and this thing is one of them.

Editors regularly complained about the fact that regular drafts and especially provisional drafts by other users (not so such revisions) are so hidden in Craft's UI, that they often ran into creating conflicting drafts, causing all sorts of disruption.

So this was brought to the sidebar in a deliberately intrusive layout

![image](https://github.com/craftcms/cms/assets/60010597/26865d81-4a07-42f5-940c-82c426419fd7)

Supporting this idea, may be recent revisions can live in a dropdown to save some space, or just a headline like '2 drafts, 4 revisions' and all the links in a dropdown

**Comment by thupsi** [comment]
I tend to agree in general with this idea. The only wrinkle is that the details sidebar is now collapsible, so the revisions could potentialy be hidden completely if they only appear there. Also, on mobile the positioning at the top of the page is more convenient. Maybe they should be in both places?

**Reply by aaronbushnell** [reply]
My problem with keeping it at the top near the breadcrumbs is that the revision list is inaccessible if the breadcrumbs are too lengthy. I've attached an example below.

I think—even with them being pushed down—it allows the revisions to have a unique visual treatment, no longer confined to the context menu like @wsydney76 is showing! 

![Screenshot 2024-03-06 at 1 19 11 PM](https://github.com/craftcms/cms/assets/315202/41ae8e2e-7541-47ab-b3bc-43674fec24ea)

**Reply by thupsi** [reply]
Aha, hadn't tested that on mobile to be honest. Looks like a bug. At least that breadcrubs bar should be scrollable.

I agree (especially after seeing that mobile layout) that it's not a great place for revisions, and a dedicated section on the sidebar would be a good solution.
